--- /usr/local/openresty/nginx/conf/nginx.conf	2020-05-14 13:27:29.168451660 -0700
+++ /usr/local/openresty/nginx/conf/nginx.conf	2020-05-14 13:28:29.583197857 -0700
@@ -24,6 +24,16 @@


 
+error_log /dev/stderr debug;
+
+env oidc_client_id;
+env oidc_client_secret;
+env oidc_discovery;
+env oidc_redirect_uri;
+env oidc_ssl_verify;
+env oidc_scope;
+env oidc_post_logout_uri;
+
 #error_log  logs/error.log;
 #error_log  logs/error.log  notice;
 #error_log  logs/error.log  info;
